spec/features/users/facebook_login_spec.rb

```diff
@@ -39,6 +39,15 @@
 
         expect(current_path).to eq("/profile")
         expect(page).to have_content("You are now logged in as Sharon")
+
+        within("#logout") do
+            click_on "logout"
+        end 
+
+        expect(current_path).to eq("/")
+        expect(page).to have_content("You are now logged out")
+        expect(page).to have_content("Login")
+        expect(page).not_to have_content("Logout")
```

Comment:
thanks for taking the time to make these tests more robust!

Reply:
![](https://media3.giphy.com/media/l0IyqqiSuoOFBdZ7i/giphy.gif?cid=5a38a5a2fw6to2uol1o76z2qlslv1vvnmhhz8su2psg09buo&rid=giphy.gif)